data/campaigns/Heir_To_The_Throne/scenarios/04_The_Bay_of_Pearls.cfg

```diff
@@ -657,6 +657,32 @@
             speaker=Heldaga
             message= _ "Death to the orcs! Come, my mer brethren, let us fight the orcs and drive them from our shores!"
         [/message]
+
+        # Give a hint about the trident.
+        [if]
+            [variable]
+                name=item_bop_stormtrident_picked_up
+                not_equals=yes
+            [/variable]
+            [then]
+                [message]
+                    speaker=Oceania
+                    message= _ "$unit.name|, we have hidden a trident in the north bay. Let us fetch it and expel the naga from these waters!"
```

Comment:
Seems clunky. Also, the Naga obviously know about it, so it's not hidden from them. So let's assume they can't touch it.

"When we were captured, one of my guards dropped his trident. Its magical powers prevent the Naga from even touching it, so it should still be in the northern bay."

It's a bit more wordy than I'd like but it blends story with the facts on the ground.

Reply:
About the backstory: Why do you say the Naga "obviously" know about the trident? I see nothing in the scenario to suggest they know it's there. FWIW, my in-world backstory was that the mermaids hid the trident either during the first battle with the orcs, when they saw the Bay was about to fall, or long before that, as a warchest for a rainy day.

Maybe we can come up with backstories for the tridents in later scenarios too.

About the phrasing of the dialog, I welcome improvements to it.

Reply:
Oh, you're correct. I remembered them guarding something. It is prisoners, not the trident. The trident just happens to be 'in range' of the guards. I like it being dropped by a guard. Oceania speaking indicates the guard is not available.

I was just thinking, that Trident, in particular, is pretty obvious. I wonder what @ln-zookeeper would say to having it not be placed on the map until Oceania and the other prisoners are released. There are a number of prisoners; one alternative would be to have the first (or second), whoever that may be, mention it and wait to place it until that point. (Downside: complete the scenario without releasing any means no Trident which is no matter because you're probably going forward with no Merfolk at all.)